cachi2/interface/cli.py

```diff
@@ -10,7 +10,7 @@
 import typer
 from typer import Argument, Option
```

Comment:
Maybe we should remove this line and just use them as `typer.Argument` and `typer.Option`. Or maybe explicitly import everything needed from the typer module.

This way we'll get rid of the CodeQL warning.

Reply:
+1 for `typer.Argument` and `typer.Option` usage

Reply:
ack, removed the extra imports